src/devices/Display/SevenSegment/category.txt

```diff
@@ -0,0 +1,2 @@
+display
```

Comment:
there should be only one category file per folder under devices. It's fine to collapse categories into a sum of all devices. Same goes for `README.md` - it should describe all devices in the folder (extra README under samples is ok)

Reply:
(technically we could allow that but that would require changing tool which generates `src/devices/README.md`)